server/webapp/WEB-INF/rails.new/webpack/views/dashboard/models/filter_validations.js

```diff
@@ -24,15 +24,33 @@ const MSG_PAD_SPACE = "Filter name must not have leading or trailing whitespace"
 const MSG_BAD_CHARS = "Filter name is only allowed to contain letters, numbers, spaces, and punctuation marks";
 const MSG_DUPE_NAME = "Another filter with this name already exists";
```

Comment:
Just a note, all the messages should change to use the word `view` instead of `filter`. does not need to happen as part of this PR though